tls/s2n_prf.c

```diff
@@ -214,6 +223,69 @@ static const struct s2n_p_hash_hmac s2n_evp_pkey_p_hash_hmac = {
     .cleanup = &s2n_evp_pkey_p_hash_cleanup,
     .free = &s2n_evp_pkey_p_hash_free,
 };
+#else
```

Comment:
Do we need these methods in an #else? Do they use symbols that aren't always available?

Reply:
In the !AWS-LC code path they are unused and cause the build to fail due unused-const-variable. The `s2n_evp_hmac_p_hash_hmac` struct only gets used in `s2n_get_hmac_implementation()` if it's AWS-LC or BoringSSL.

Reply:
Gotcha. And if you make only the struct part of the else, all the static methods won't be referenced. Maybe make the structure not static?

Reply:
I stand corrected, these functions exist in OpenSSL 1.1.1 but not FIPS 1.0.2. I think the simpler if/else is marginally cleaner than a if/else if which then special cases two different things.

Reply:
Definitely :) We'd only move them out of the else if they could always be compiled.